tests/containers/nccl-tests/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG CUDA_VERSION_MINOR=12.8.1
```

Comment:
We might want to double check the CUDA version that is part of GPU Operator / AKS managed VHD by running `nvidia-smi` in a test pod. I don't think it supports CUDA 12.8 yet.

Reply:
I think RDMA won't work there because of the peermem module.

Reply:
Just checked that the latest GPU operator supports CUDA 12.8.